features/step_definitions/assertion_steps.rb

```diff
@@ -1,3 +1,7 @@
 When("I am at the index page") do
     visit root_path
+end
+
+Then("I should be on the Landing page") do
+    expect(current_path).to eq root_path
```

Comment:
What is the difference between landing and index page in this case..?

Reply:
The Landing page and the index is the same thing but feels more in scope referring to the fist page of the application. The first page the user is presented with.